public/src/components/channelManagement/epicTests/epicTestEditor.tsx

```diff
@@ -207,23 +201,14 @@ export const getEpicTestEditor = (
 
     return (
       <div className={classes.container}>
-        <div className={classes.switchContainer}>
-          <LiveSwitch
-            label="Live on Guardian.com"
-            isLive={test.isOn}
-            isDisabled={!userHasTestLocked}
-            onChange={onLiveSwitchChange}
-          />
-          <div>
-            <EpicTestPreviewButton test={test} />
-          </div>
-        </div>
-
         {epicEditorConfig.allowMultipleVariants && (
           <div className={classes.sectionContainer}>
-            <Typography variant={'h3'} className={classes.sectionHeader}>
-              Variants
-            </Typography>
+            <div className={classes.variantsHeaderContainer}>
+              <Typography variant={'h3'} className={classes.sectionHeader}>
+                Variants
+              </Typography>
+              <EpicTestPreviewButton test={test} />
```

Comment:
this change moves the `EpicTestPreviewButton` button in-line with the "Variants" heading, because it used to be in-line with the status toggle